core/services/assets/BlockAssetManager.php

```diff
@@ -136,8 +136,8 @@ public function addAssets()
     {
         $this->addEditorScript($this->getEditorScriptHandle());
         $this->addEditorStyle($this->getEditorStyleHandle());
-        $this->setScriptHandle($this->getScriptHandle());
-        $this->setStyleHandle($this->getStyleHandle());
```

Comment:
uhhh... we still need to set the script and style handles else things will break!!!

for example:

- `BlockAssetManager::getScript()` references `$this->script_handle` which will no longer be set!
- `BlockAssetManager::getStyle()` references `$this->style_handle` which will no longer be set!
- `\EventEspresso\core\domain\entities\editor\Block::registerBlock()` references the getters for both of the above properties!!!
- you now have `addDefaultBlockScriptDependencies()` calling `getScriptHandle()` but `$this->script_handle` is no longer be set!
- you have added `addDefaultBlockStyleDependencies()` which calls `getStyleHandle()` but `$this->style_handle` is no longer be set!


Why exactly was this done?

and more importantly... how is this not blowing up for you when you try to use things? I can only guess that things have changed so blinking much with GB that all of this asset manager and block registration stuff is working completely differently now.

Reply:
1. If that was the case then why does `addEditorScript` and `addEditorStyle` not have a `setEditorScriptHandle` and `setEditorStyleHandle` call?
2. `setScriptHandle` and `setScriptStyle` are actually called in the child class (`CoreBlocksAssetManager`)

```
/**
     * @since $VID:$
     */
    public function setAssetHandles()
    {
        $this->setEditorScriptHandle(self::JS_HANDLE_CORE_BLOCKS_EDITOR);
        $this->setEditorStyleHandle(self::CSS_HANDLE_CORE_BLOCKS_EDITOR);
        $this->setScriptHandle(self::JS_HANDLE_CORE_BLOCKS);
        $this->setStyleHandle(self::CSS_HANDLE_CORE_BLOCKS);
    }
```

This was done because nothing was actually *adding* the script and styles and thus things WERE broken.  The work I did is the first time the frontend script/styles were actually tested so...

Reply:
right... sry forgot that asset handles were set there now... it wasn't like that initially but ended up changing.


so ya the change you made here was likely something I missed when handle setting was moved to the child classes.